pkg/tfvars/internal/cache/cache.go

```diff
@@ -22,6 +23,7 @@ const (
 	applicationName = "openshift-installer"
 	imageDataType   = "image"
 	gzipFileType    = "application/x-gzip"
+	xzFileType      = "application/octet-stream"
```

Comment:
Don't we want to be more specific and use `application/x-xz`?

```
$ file fedora-coreos-31.20200118.2.0-openstack.x86_64.qcow2.xz -i
fedora-coreos-31.20200118.2.0-openstack.x86_64.qcow2.xz: application/x-xz; charset=binary
```

Reply:
FCOS server would only send us `octet-stream` unfortunately